models/vista3d/configs/data.yaml

```diff
@@ -0,0 +1,39 @@
+# this file shows specific settings for evaluation on each dataset
+validate#postprocessing#transforms#0#_disabled_: false
+validate#handlers#2#_disabled_: true
+validate#evaluator#key_val_metric: null
+eval_folder: "/eval/"
+ckpt_path: "$@bundle_root + '/models/model.pt'"
+label_set: "$list(set([i+1 for i in range(117)]) - set([22, 23, 15, 25, 19, 2, 26, 27, 28, 29, 117]))"
+val_label_set: "$list(range(118))"
+label_mappings: "${}"
+# label_mappings:
```

Comment:
May I ask why we comment the default mapping here?

Reply:
It comes from the author side, I did change here.